test/wdt_e2e_simple_test.sh

```diff
@@ -137,7 +137,13 @@ if [ $DO_VERIFY -eq 1 ] ; then
     echo "Should be no diff"
     (cd $DIR; diff -u src.md5s dst.md5s)
     STATUS=$?
+    if [ $STATUS -eq 0 ] ; then
+      (cd $DIR/src ; ( find . -type f -printf %f%m\\n | sort ) > ../src.perm )
```

Comment:
1. OK. I will add a environment var for that. Sorry but I do not have a mac at hand...

2. Well... there are...
If I just change this line https://github.com/facebook/wdt/pull/155/files#diff-a77a3101b61bf7da9963a0791dba5bb6R247 to return 0644, 
The following tests FAILED:
	 10 - FileReaderTests (Failed)
	 12 - WdtBasicE2E (Failed)
	 19 - WdtSimpleOdirectTest (Failed)
For 12 & 19, we have things like this:
Should be no diff
--- src.perm	2017-04-05 14:56:37.893425997 -0700
+++ dst.perm	2017-04-05 14:56:37.893425997 -0700
@@ -1,7 +1,7 @@
-a664
-c664
-file1664
-file1664
+a644
+c644
+file1644
+file1644

(Part of "ls -l" output on the src dir is here.)
![screenshot from 2017-04-05 15-05-34](https://cloud.githubusercontent.com/assets/24802714/24729161/c069c830-1a11-11e7-8f3e-c29128d5576f.png)